banzai_floyds/tests/test_background.py

```diff
@@ -0,0 +1,96 @@
+from banzai_floyds.background import fit_background, set_background_region, BackgroundFitter
```

Comment:
Mostly a refactor. We now test against all pixels in the order except the outer edge (2 pixels wide). Due to the interpolation limitations, these pixels can be bad, but given they are away from the extraction, we don't really mind.